src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -2886,6 +2892,10 @@ components:
           $ref: '#/components/schemas/DatasetSpecificationModel'
         region:
           type: string
+        gcpRegion:
+          type: string
+          description: >
+            This is a temporary parameter until GCP and Azure resources are no longer entwined.  This parameter will disappear shortly
```

Comment:
Can you expand a bit more on why we need this parameter? Should we instead break the azure region into its own parameter?

Reply:
Absolutely.  So, the gcpParameter is a temporary one.  Eventually, when all of the resources are in azure or gcp (but not both) then we can get rid of this parameter and just use the generic `region` parameter.  This is to tide us over until then (with the expectation that it will get removed, so no one is surprised when it is)